cmd/tether/attach.go

```diff
@@ -382,6 +387,16 @@ func (t *attachServerSSH) channelMux(in <-chan *ssh.Request, process *os.Process
 					log.Errorf("Failed to dispatch signal to process: %s\n", err)
 				}
 			}
+		case msgs.CloseStdinReq:
+			msg := msgs.CloseStdinMsg{}
```

Comment:
There's no data sent over this channel so we can drop the unmarshalling and structure definitions.